core/src/main/java/org/htsjdk/core/utils/IOUtils.java

```diff
@@ -0,0 +1,21 @@
+package org.htsjdk.core.utils;
+
+import java.io.File;
```

Comment:
No `File` in htsjdk3. This class should be removed and the only method usages modified in favor of [Files.createTempFile](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempFile(java.lang.String,%20java.lang.String,%20java.nio.file.attribute.FileAttribute...))

Reply:
Good point - done.